wiki/Beatmap/Countdown/de.md

```diff
@@ -0,0 +1,17 @@
+---
+tags:
+  - timer
+  - Count-down
+---
+
+# Countdown
+
+*Für Turniertimer, siehe: [Befehle für das Turnier-Management](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)*\
+*Für Vorschriften zu Countdowns, siehe: [Rankingkriterien § Allgemein § Richtlinien](/wiki/Ranking_criteria#guidelines)*\
+*Siehe auch: [Offset (Begriffsklärung)](/wiki/Offset)*
+
+Ein **Countdown** kann am Anfang einer [Beatmap](/wiki/Beatmap) erscheinen. Die meisten Beatmaps benötigen keinen Countdown, bei schnell oder abrupt startenden Songs kann er aber nützlich sein. Im `Design`-Tab des [Song-Setup-Fensters](/wiki/Client/Beatmap_editor/Song_setup) kann der Countdown an- und ausgeschaltet werden. In [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps darf kein Countdown verwendet werden.
+
+Die Countdown-Animation kann mithilfe der Buttons zur `Countdown Speed` (`Half`, `Normal`, `Double`) und mithilfe des Eingabefelds `Countdown Offset` angepasst werden. Die Buttons unter `Countdown Speed` können die Standardgeschwindigkeit der Countdown-Animation halbieren, beibehalten oder verdoppeln. Der Wert in `Countdown Offset` legt die Anzahl an zusätzlichen [Grundschlägen](/wiki/Music_theory/Tempo) fest, die zwischen dem Ende des Countdowns und dem ersten [Hit-Objekt](/wiki/Gameplay/Hit_object) eingefügt werden. Zum Beispiel startet bei einem `Countdown Offset` von 2 die Countdown-Animation 2 Grundschläge früher als sonst. Bei Maps mit Intros führt das Überspringen des Intros direkt zum Countdown.
```

Comment:
```suggestion
Die Countdown-Animation kann mithilfe der Buttons zur `Countdown Speed` (`Half`, `Normal`, `Double`) und mithilfe des Eingabefelds `Countdown Offset` angepasst werden. Die Buttons unter `Countdown Speed` können die Standardgeschwindigkeit der Countdown-Animation halbieren, beibehalten oder verdoppeln. Der Wert in `Countdown Offset` legt die Anzahl an zusätzlichen [Grundschlägen](/wiki/Music_theory/Tempo) fest, die zwischen dem Ende des Countdowns und dem ersten [Hit-Objekt](/wiki/Gameplay/Hit_object) eingefügt werden. Zum Beispiel startet bei einem `Countdown Offset` von 2 die Countdown-Animation 2 Grundschläge früher als sonst. Bei Beatmaps mit Intros führt das Überspringen des Intros direkt zum Countdown.
```
I know that `Map` is pretty common, however, I have been avoiding this in the wiki and always used `Beatmap` instead.